user-console/src/main/resources/org/pentaho/mantle/public/browser/js/browser.js

```diff
@@ -99,7 +99,11 @@ define([
 
   FileBrowser.encodeGenericFilePathComponents = function (path) {
     const encodedFilePath = Encoder.encodeGenericFilePath(path);
-    return Encoder.encode("{0}", encodedFilePath);
+    return FileBrowser.encodePathComponents(encodedFilePath);
+  }
+
+  FileBrowser.encodeURLParameterValue = function (value) {
```

Comment:
add JsDOc for FileBrowser.encodeURLParameterValue, it may be just to reference where that Encoder is defined in https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/impl/client/src/main/javascript/web/util/URLEncoder.js

Reply:
my comment seems obsolete since change of approach with https://github.com/pentaho/pentaho-platform/pull/5593/commits/5e72249baf7b9ffaa9d493486f33a7e5ab4d4ca5